src/utils.js

```diff
@@ -23,6 +23,16 @@ export function safelyRegisterModule(store, name, vuexModule, logger) {
     }
 }
 
+/**
+ * Return an array of matched components for the given route
+ *
+ * @param   {Object} route VueRouter route instance
+ * @returns {Object[]}    Array of matched components
+ */
+export function getMatchedComponents(route) {
+    return route.matched.flatMap(r => Object.values(r.components));
```

Comment:
Suggested replacement from https://next.router.vuejs.org/guide/migration/#removal-of-router-getmatchedcomponents